boards/arm/mimxrt1010_evk/mimxrt1010_evk.mex

```diff
@@ -0,0 +1,1060 @@
+<?xml version="1.0" encoding= "UTF-8" ?>
+<configuration name="MIMXRT1011xxxxx" xsi:schemaLocation="http://mcuxpresso.nxp.com/XSD/mex_configuration_10 http://mcuxpresso.nxp.com/XSD/mex_configuration_10.xsd" uuid="de3bdbe5-65cf-482d-b1be-cd17b780459b" version="10" xmlns="http://mcuxpresso.nxp.com/XSD/mex_configuration_10" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
IMO adding those files should be part of commits that include `clock_config.h` (which is generated from those `.mex` files).